Framework/PythonInterface/plugins/algorithms/WorkflowAlgorithms/SANSILLMultiProcess.py

```diff
@@ -0,0 +1,990 @@
+        self.setPropertyGroup('SampleThickness', 'Parameters')
+
+        self.declareProperty(name='SampleThicknessFrom', defaultValue='User',
+                             validator=StringListValidator(['User', 'Nexus']),
+                             doc='Define where to read the sample thicknesses from.')
+        self.setPropertyGroup('SampleThicknessFrom', 'Parameters')
+
+        self.declareProperty(StringArrayProperty(name='SampleNames', values=[]),
+                             doc='Sample names to put in the axis of the output workspaces.')
+        self.setPropertyGroup('SampleNames', 'Parameters')
+
+        self.declareProperty(name='SampleNamesFrom', defaultValue='RunNumber',
+                             validator=StringListValidator(['User', 'Nexus', 'RunNumber']),
+                             doc='Define where to read the sample names from.')
+        self.setPropertyGroup('SampleNamesFrom', 'Parameters')
+
+        self.declareProperty(name='WaterCrossSection', defaultValue=1.,
+                             validator=FloatBoundedValidator(lower=0.),
+                             doc='Provide the water cross-section; used only if the absolute scale is done by dividing to water.')
+        self.setPropertyGroup('WaterCrossSection', 'Parameters')
+
+        self.declareProperty(name='ProduceSensitivity', defaultValue=False,
+                             doc='Whether or not to produce a sensitivity map; should be used for water reduction only.')
+        self.setPropertyGroup('ProduceSensitivity', 'Parameters')
+
+        self.declareProperty(name='SensitivityWithOffsets', defaultValue=False,
+                             doc='Whether the sensitivity data has been measured with different horizontal offsets (D22 only).')
+        self.setPropertyGroup('SensitivityWithOffsets', 'Parameters')
+
+        self.declareProperty(name='OutputType', defaultValue='I(Q)',
+                             validator=StringListValidator(['None', 'I(Q)']),
+                             doc='The type of the integration to perform.')
+        self.setPropertyGroup('OutputType', 'Parameters')
+
+        self.declareProperty(IntArrayProperty(name='DistancesAtWavelength2', values=[]),
+                             doc='Defines which distance indices (starting from 0) match to the 2nd wavelength')
+        self.setPropertyGroup('DistancesAtWavelength2', 'Parameters')
+
+        self.declareProperty(name='SubAlgorithmOffset', defaultValue=True,
+                             doc='Whether to offset logging from child algorithms one level down.')
+        self.setPropertyGroup('SubAlgorithmOffset', 'Parameters')
+
+        #===================================I(Q) OPTIONS==================================#
+
+        self.declareProperty(name='OutputBinning', defaultValue='',
+                             doc='Output binning for each distance( : separated list of binning params).')
```

Comment:
If OutputType=None do any of these I(Q) options apply? If no then could they be disabled?

Reply:
That flag was a leftover, I removed that.